tasker/android.js

```diff
@@ -35,9 +35,11 @@ var getAndroidDevices = function () {
   if (res[0].indexOf("List of devices") == 0) {
     for (var i = 1; i < res.length; i++) {
       if (res[i].trim().length == 0) continue;
-      if (res[i].indexOf('offline') > 0) {
-        logme("Warning: Phone " + res[i] + " OFFLINE");
-        continue; // phone offline
+      if (res[i].indexOf('offline') > 0 ||
+          res[i].indexOf('unauthorized') > 0 ||
```

Comment:
`unauthorized` means, we did not enabled adb permission on the phone. Since this is not a random error or warning, I believe test shouldn't continue when the test system itself broken with an unauthorized device.